src/components/InstalledKonnectors.jsx

```diff
@@ -18,11 +18,11 @@ import AccountPicker from './AccountPicker'
 import addAccountIcon from '../assets/icons/icon-plus.svg'
 import pictureForEmtpyList from '../assets/images/connected-accounts.svg'
 
-const ButtonLinkToStore = ({ icon, label, cozyStoreURL }) => (
-  <ButtonLink disabled={!cozyStoreURL} href={cozyStoreURL}>
+const StoreButton = ({ icon, label, handleClick }) => (
+  <Button onClick={handleClick}>
```

Comment:
What happen if I double-click ? Is the intent created twice ?

Is it possible to disable the button after a click here ?

As I do not see any specific code related to the store, maybe we may make this component more abstract ? What do you think ?

Reply:
I will add a state to disable the button and add a spinner but it should be a separate event for the double click anyways.